src/stage2/Parse.zig

```diff
@@ -2963,7 +2963,14 @@ fn parsePrimaryTypeExpr(p: *Parse) !Node.Index {
                     },
                 }
             },
-            else => return null_node,
+            else => return p.addNode(.{
+                .tag = .unreachable_literal,
+                .main_token = p.tok_i,
+                .data = .{
+                    .lhs = undefined,
+                    .rhs = undefined,
+                },
+            }),
```

Comment:
What is the motivation for this change? I undid this change and ran tests and tried you example code from the PR description and everything was still working.

Reply:
I had placebo'd myself, but had already reverted it as part of the "Take out the guesswork and use a debugger" changes that I just pushed.